frontend/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
+# testing
+/coverage
+
+# production
+/build
```

Comment:
I think it's not needed by the front... it's already in main `.gitignore`

Reply:
Let's please keep the conversation in one thread, above.